src/main/java/org/terasology/lost/generator/LostWorldGenerator.java

```diff
@@ -90,47 +89,70 @@ protected WorldBuilder createWorld() {
                 .addRasterizer(new WhittakerRasterizer())
                 .addRasterizer(new RiverRasterizer())
                 .addRasterizer(new TreeRasterizer())
-                .addRasterizer(new FloraRasterizer());
-//                .addPlugins();
+                .addRasterizer(new FloraRasterizer())
+                .addPlugins();
     }
 
     @Override
     public Vector3f getSpawnPosition(EntityRef entity) {
         LocationComponent loc = entity.getComponent(LocationComponent.class);
         Vector3f pos = loc.getWorldPosition();
-
-        int searchRadius = 500;
-        Vector3i ext = new Vector3i(searchRadius, 1, searchRadius);
+        Vector3i ext = new Vector3i(SEARCH_RADIUS, 1, SEARCH_RADIUS);
         Vector3i desiredPos = new Vector3i(pos.getX(), 1, pos.getZ());
 
         // try and find somewhere in this region a spot to land
-        Region3i spawnArea = Region3i.createFromCenterExtents(desiredPos, ext);
-        Region worldRegion = getWorld().getWorldData(spawnArea);
+        Region3i searchArea = Region3i.createFromCenterExtents(desiredPos, ext);
+        org.terasology.world.generation.Region worldRegion = getWorld().getWorldData(searchArea);
```

Comment:
why is the full path required?

Reply:
Becuase we have two classes named Region one in Polyworld module and one in the main world generation module.

Reply:
I will rename the one in Polyworld once this PR is merged

Reply:
Okay, then please either start a PR for that in Polyworld already or write an issue for it, so it doesn't get lost :upside_down_face:

Reply:
Was this done?